src/Affjax/Driver/Node.purs

```diff
@@ -0,0 +1,85 @@
+module Affjax.Driver.Node
+  ( request
+  , get
+  , post
+  , post_
+  , put
+  , put_
+  , delete
+  , delete_
+  , patch
+  , patch_
+  ) where
+
+import Prelude
+
+import Affjax (Error, Request, Response, URL)
+import Affjax.Driver as AD
+import Affjax.Driver (AffjaxDriver)
+import Affjax.RequestBody as RequestBody
+import Affjax.ResponseFormat as ResponseFormat
+import Data.Either (Either)
+import Data.Maybe (Maybe)
+import Effect.Aff (Aff)
+
+foreign import nodeDriver :: AffjaxDriver
```

Comment:
I'd suggest we just call these `driver` too when exporting, they can be distinguished by module name.